swagger/auth/auth.swagger.json

```diff
@@ -23,8 +28,109 @@
   "produces": [
     "application/json"
   ],
-  "paths": {},
+  "paths": {
+    "/auth/v1/method/token": {
+      "put": {
```

Comment:
should this be a `put` or a `post`? im wondering if this is idempotent or not?

Reply:
Mmm good shout. This should be a `POST`. It is non-idempotent.